src/FermentableDialog.cpp

```diff
@@ -47,14 +49,16 @@ FermentableDialog::FermentableDialog(MainWindow* parent) :
    tableWidget->setModel(fermTableProxy);
    tableWidget->setSortingEnabled(true);
    tableWidget->sortByColumn( FERMNAMECOL, Qt::AscendingOrder );
+   tableWidget->verticalHeader()->setDefaultSectionSize(20);
```

Comment:
I have removed the line as requested.
I will look into the problem with the resizing. Sorry for not beeing active but my internet connection broke down. It took the ISP several days to gt it back up.